java/vector/src/test/java/org/apache/arrow/vector/file/TestArrowFile.java

```diff
@@ -107,10 +107,9 @@ public void testWriteRead() throws IOException {
       try (VectorSchemaRoot root = new VectorSchemaRoot(schema, vectorAllocator)) {
         VectorLoader vectorLoader = new VectorLoader(root);
 
-        List<ArrowBlock> recordBatches = footer.getRecordBatches();
-        for (ArrowBlock rbBlock : recordBatches) {
-          Assert.assertEquals(0, rbBlock.getOffset() % 8);
-          Assert.assertEquals(0, rbBlock.getMetadataLength() % 8);
+        List<Integer> recordBatches = footer.getRecordBatches();
+        for (Integer rbBlock : recordBatches) {
+          //Assert.assertEquals(0, rbBlock % 8);
```

Comment:
Only the starts of buffers need to be aligned (the buffers are already supposed to be padded). So only need to add padding to the serialized metadata